CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
 
 #### New Features
 
+#### Improvements
+- Disables sql simplification when sort is performed after limit. 
+  - Respect the original clause order, before `df.sort().limit()` and `df.limit().sort()` are the same as `df.sort().limit()`.
```

Comment:
I understand what you mean here is that before this change, we should treat `df.sort().limit()` the same as `df.limit().sort()` but with this change, we fix it. @sfc-gh-pspaeth can you suggest how we can update this changelog.

Reply:
updated based on the feedback